content/docs/cucumber/api.md

```diff
@@ -57,9 +73,12 @@ Given("the following animals:") { animals: java.util.List[String] =>
 ```
 {{% /text %}}
 
-In this case, the `DataTable` is automatically flattened to a {{% text "java,kotlin" %}}`List<String>`{{% /text %}}{{% text "scala" %}}`java.util.List[String]`{{% /text %}}
+In this case, the `DataTable` is automatically flattened to a {{% text "java,kotlin" %}}`List<String>`{{% /text %}}{{% text "scala" %}}`java.util.List[String]`{{% /text %}}{{% text "ruby,javascript" %}}list of strings{{% /text %}}
 by Cucumber (using `DataTable.asList(String.class)`) before invoking the step definition.
 
+{{% text "java,kotlin" %}}Note: In addition to collections of String, Integer, Float, BigInteger and BigDecimal, Byte,
```

Comment:
Minor: What is also supported, is that _lists_ of those data types, _or_ those data types themselves?

Reply:
Collections include lists